<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Polygons drawing ICMC-USP</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="DrawingArea" name="drawingAreaWidget" native="true"/>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QSpinBox" name="sidesSpinBox">
         <property name="accessibleDescription">
          <string/>
         </property>
         <property name="minimum">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="radiusSpinBox">
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="drawPushButton">
         <property name="text">
          <string>Draw</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearPushButton">
         <property name="text">
          <string>Crear</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>908</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DrawingArea</class>
   <extends>QWidget</extends>
   <header location="global">drawingarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
